bin/handler-microsoft-teams.rb

```diff
@@ -123,14 +123,14 @@ def build_description
     template = if message_template && File.readable?(message_template)
                  File.read(message_template)
                else
-                 '''<%=
+                 '<%=
```

Comment:
In ruby there are no triple magic quotes like python. Here is a demonstration showing they are exactly the same as a regular quote:
```
$ irb
irb(main):001:0> a = '<%=
irb(main):002:0'                  [
irb(main):003:0'                    @event["check"]["output"].gsub(\'"\', \'\\"\'),
irb(main):004:0'                    @event["client"]["address"],
irb(main):005:0'                    @event["client"]["subscriptions"].join(",")
irb(main):006:0'                  ].join(" : ")
irb(main):007:0'                  %>
irb(main):008:0'                  '
=> "<%=\n                 [\n                   @event[\"check\"][\"output\"].gsub('\"', '\\\"'),\n                   @event[\"client\"][\"address\"],\n                   @event[\"client\"][\"subscriptions\"].join(\",\")\n                 ].join(\" : \")\n                 %>\n                 "
irb(main):009:0> a
=> "<%=\n                 [\n                   @event[\"check\"][\"output\"].gsub('\"', '\\\"'),\n                   @event[\"client\"][\"address\"],\n                   @event[\"client\"][\"subscriptions\"].join(\",\")\n                 ].join(\" : \")\n                 %>\n                 "
irb(main):010:0> b = '''<%=
irb(main):011:0'                  [
irb(main):012:0'                    @event["check"]["output"].gsub(\'"\', \'\\"\'),
irb(main):013:0'                    @event["client"]["address"],
irb(main):014:0'                    @event["client"]["subscriptions"].join(",")
irb(main):015:0'                  ].join(" : ")
irb(main):016:0'                  %>
irb(main):017:0'                  '''
=> "<%=\n                 [\n                   @event[\"check\"][\"output\"].gsub('\"', '\\\"'),\n                   @event[\"client\"][\"address\"],\n                   @event[\"client\"][\"subscriptions\"].join(\",\")\n                 ].join(\" : \")\n                 %>\n                 "
irb(main):018:0> if a == b then p 'a == b' else p 'a != b' end
"a == b"
=> "a == b"
```

I removed the triple quotes to avoid the implicit string concatination of `''` (empty string) and `'<%=...`